eldritch/encounters.py

```diff
@@ -244,8 +244,97 @@ def Science7(char):
   helping = events.PassFail(char, check, close_gates, fail)
   return events.BinaryChoice(char, "Offer to help?", "Yes", "No", helping, events.Nothing())
 
+def Shoppe1(char):
```

Comment:
Note: the curiositie shoppe is named "Shop". Þe old magick shoppe is named "Shoppe". If that's confusing, you can feel free to rename them.

Reply:
ah, okay, will rename everything

Reply:
everything has been renamed.